src/App.js

```diff
@@ -1,12 +1,58 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
+import axios from 'axios';
 import './App.css';
+import PokemonCard from "./components/PokemonCard"
 
 function App() {
-  return (
-    <div>
-      Begin hier met de opdracht!
-    </div>
-  );
+    const [pokemonData, setPokemonData] = useState([]);
```

Comment:
Op regel 14 zet je de waarde, maar dat is geen array. Daarom zou ik dus niet als default value voor een lege array gaan. Ik zou kiezen voor `undefined` zodat je op regel 25 daar een goeie check op kunt doen. De default value moet eigenlijk altijd van hetzelfde type zijn als waarmee je setPokemonData mee invoked.